sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -829,7 +829,7 @@ class CastSuite extends SparkFunSuite with ExpressionEvalHelper {
   }
 
   test("SPARK-22500: cast for struct should not generate codes beyond 64KB") {
-    val N = 250
+    val N = 25
```

Comment:
is 25 big enough to reproduce the bug of SPARK-22595?

Reply:
Yes, I confirmed that 25 can reproduce the bug of SPARK-22500 by reverting the change of #19730. For example, 20 cannot reproduce the bug.